### ISSUE TITLE
Basic shape icons?

### ISSUE THREAD

**Issue by mxo9722** [game-icons/icons#971]
You have a circle and square but you don't seem to have a default triangle which is disappointing

**Comment by Delapouite** [comment]
Yes, the reasoning behind this kind of absence is that this adds no real value to the collection as a triangle can be done by anyone.

**Comment by aloisdg** [comment]
Because a square and a circle are especially harder to build? As a complete neophyte, a triangle or a pentagon could be useful.

**Comment by Rdjrjqouqcu** [comment]
I've been looking for a set of basic shape icons, and it just doesn't seem to be a thing that exists anywhere. I think just having a sprite sheet that contains a permutation of everything would be great, despite how anyone can make them.

I'd like to suggest adding a new tag for `basic_shapes` that would include the following icons:

Currently existing:
- [Circle](https://game-icons.net/1x1/delapouite/circle.html)
- [Square](https://game-icons.net/1x1/delapouite/square.html)
- [Plain Circle](https://game-icons.net/1x1/delapouite/plain-circle.html)
- [Plain Square](https://game-icons.net/1x1/delapouite/plain-square.html)
- [Plain Arrow](https://game-icons.net/1x1/delapouite/plain-arrow.html)
- [5 Pointed Rounded Star](https://game-icons.net/1x1/delapouite/round-star.html)
- [7 Pointed Star](https://game-icons.net/1x1/delapouite/seven-pointed-star.html)

From the above images I think the following would be useful:
- All shapes are either Plain (solid) or just the outline
- All shapes are either Rounded or just the sharp corners

So the family of square shapes would be "Square", "Plain Square", "Rounded Square", "Plain Rounded Square"

I'm mainly thinking the following would be useful. Anything else that seems to fit can probably be added later.

- Circle
- Regular Polygons: 3 sides to 12 sides
  - i.e. Triangle, Square, Pentagon, Hexagon, etc.
  - [image](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEg3C8rEbDJMxQE3-eHClMHiV-syTqvF3C1uB75SUyl1JjSr4xHsl9_sEarB3PqNdC-ESJ-9FW7_jhq5LJzGj8DW859DBgwd11IQ14qbWf2JlUa6M9N9r5g5Ua28pGS6VZ4fjdJS71e-mQ/s1600/geo12.jpg)
- Pointed Stars: 3 sides to 12 sides
  - [image](https://www.vectortemplates.com/bincdn2/files/editor_images/stars-0111.png)
- Arrow shape
  - [image](https://game-icons.net/1x1/delapouite/plain-arrow.html)
